protecode/model.py

```diff
@@ -85,6 +88,10 @@ def __repr__(self):
             f'{self.version() or "Version not detected"}'
         )
 
+    def __iter__(self):
```

Comment:
why?

Reply:
The `dict()` constructor accepts an iterable of `(key, value)` pairs to construct a dictionary.
Implementing the `__iter__` enables simple dict parsing from class, in order to store scan_result in compliance db.